lab_1_classify_by_unigrams/main.py

```diff
@@ -11,6 +12,12 @@ def tokenize(text: str) -> list[str] | None:
     :param text: a text
     :return: a list of lower-cased tokens without punctuation
     """
+    if not isinstance(text, str):
+        return None
+
+    tokens = [character for character in text.lower() if character.isalpha()]
```

Comment:
perfect, lets remove it without creating the variable

Reply:
Did it. Hopefully, in the way you meant it